src/backend/linalg_internal_gpu/cuTensordot_internal.cu

```diff
@@ -141,8 +141,7 @@ namespace cytnx {
        ***************************/
 
       cutensorPlanPreference_t planPref;
-      cutensorCreatePlanPreference(handle, &planPref, CUTENSOR_ALGO_DEFAULT,
```

Comment:
Does this improve the performance in all cases? Is there any drawback of picking a specific algorithm?

Reply:
I'm not sure if that applies to all cases, but the performance improvement is pretty large in the case of chi=16, D=4 CTMRG.